rq/worker.py

```diff
@@ -690,10 +690,12 @@ def dequeue_job_and_maintain_ttl(self, timeout):
                 if self.should_run_maintenance_tasks:
                     self.run_maintenance_tasks()
 
+                self.log.debug(f"Dequeing jobs on queues {self._ordered_queues} and timeout {timeout}")
```

Comment:
Sorry, minor nitpick `Dequeueing job from self._ordered_queues with timeout of {}`